LayoutTests/inspector/css/getMatchedStylesForNodeNestingStyleGrouping.html

```diff
@@ -112,6 +147,16 @@
     #outerA {
         color: red;
 
+        @media(min-width: 0px) {
+            background-color: blue;
+            
+            & {
+                color: yellow;
+            }
+
+            color: purple;
```

Comment:
ooohhh so `isImplicitlyNested` covers cases like this (i.e. the same selector, but a different `@rule`)?  i wonder if maybe we could name it something like `isNestedAtRule` to clarify that instead 🤔

Reply:
isNestedAtRule is less clear IMO since the styles are not in an @rule according to CSSOM, but are instead implicitly nested as a style rule inside the @rule.